pkg/cmd/server/origin/ensure.go

```diff
@@ -151,7 +153,47 @@ func (c *MasterConfig) ensureNamespaceServiceAccountRoleBindings(namespace *kapi
 	}
 }
 
+func (c *MasterConfig) securityContextConstraintsSupported() (bool, error) {
```

Comment:
Add TODO to make this a library upstream, `ResourceExists(GroupVersionResource)` or some such.  Definitely not a blocker, just seen a few these ripple through.